e2e/keywords/host.resource

```diff
@@ -73,3 +73,13 @@ Power off node ${node_id}
     ${powered_off_node} =     get_node_by_index    ${node_id}
     Append to list      ${powered_off_nodes}     ${powered_off_node}
     power_off_node_by_name    ${powered_off_node}
+
+Create snapshot on node ${node_id}
+    ${node_name} =     get_node_by_index    ${node_id}
+    create_snapshot    ${node_name}
+
+Create snapshot on volume node of ${workload_kind} ${workload_id}
```

Comment:
Is it possible to update the keyword `Create snapshot  ***` in this file to indicate that it refers to a VM snapshot, since most test cases use `Create snapshot` to refer to a volume snapshot, beside this no other quesion and test case passed with [v1](https://ci.longhorn.io/job/private/job/longhorn-e2e-test/3218/), [v2](https://ci.longhorn.io/job/private/job/longhorn-e2e-test/3219/) data engine

Reply:
Refined. Thank you!

Reply:
Hi @yangchiu 
     I noticed that after the merge of https://github.com/longhorn/longhorn-tests/pull/2500 , `Pre-release Checks` fav started to fail. Could you please take a look when you have a moment?

![Image](https://github.com/user-attachments/assets/d9d7114d-0ebb-4311-8b31-890e31085b23)

https://ci.longhorn.io/job/private/job/longhorn-helm-chart-test/129/robot/report/log.html